src/Kmd.Logic.Gateway.Automation/Gateway/ApiVersion.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Kmd.Logic.Gateway.Automation.Gateway
+{
+    public class ApiVersion
+    {
+        public string PathIdentifier { get; set; }
+
+        public string ApiLogoFile { get; set; }
+
+        public string ApiDocumentation { get; set; }
+
+        public string OpenApiSpecFile { get; set; }
+
+        public string Published { get; set; }
+
+#pragma warning disable CA1819 // Properties should not return arrays
```

Comment:
Instead of disabling warning could you change it to IReadOnlyList<string>. And the same for other similar errors?
I think it make sense to unify way of storing collection so we don't end with one property as array and another as list.

Reply:
Yeah, here should be IReadOnlyList or IEnumerable and I think the error disappears

Reply:
IEnumerable on property is never good idea

Reply:
IReadonly throw exception while deserialization, I have changed to IEnumerable